ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -430,15 +430,18 @@ public struct Avatar: View, TokenizedControlView {
             return Int(abs(javaHashCode(combinedHashable)))
         }
 
-        /// To ensure iOS and Android achieve the same result when generating string hash codes (e.g. to determine avatar colors) we've copied Java's String implementation of `hashCode`.
+        /// Hash algorithm to determine Avatar color.
+        /// Referenced from: https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-avatar/src/components/Avatar/useAvatar.tsx#L200
         /// Must use Int32 as JVM specification is 32-bits for ints
         /// - Returns: hash code of string
         private static func javaHashCode(_ text: NSString) -> Int32 {
```

Comment:
I think "java hash code" refers to the old algorithm, might want to rename this to "webHashCode"

Reply:
Thinking of changing it to just `hashCode` actually, would that sound ok?

Reply:
Sounds good! You should probably also remove the reference to `JVM`, since that was the Java Virtual Machine from the old algorithm too.